src/main/java/com/bringframework/exceptions/BeanDefinitionDuplicateNameException.java

```diff
@@ -0,0 +1,15 @@
+package com.bringframework.exceptions;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+
+/**
+ * Exception to indicate that a {@link BeanDefinition} with the same name already exists in the
+ * {@link BeanDefinitionRegistry}.
+ **/
```

Comment:
There are exceptions already in our project, so it would be great to align javadoc across all these classes, so the they have more or less the same sentence structure, etc